docs/guides/components.md

```diff
@@ -55,12 +110,180 @@ Results in a DOM that looks like this:
 </div>
 ```
 
-Again, refer to [the component API docs](http://docs.videojs.com/docs/api/component.html) for complete details on methods available for managing component structures.
+### Using Opitons
```

Comment:
Seems like this is a new section altogether. Can we pull additions into a separate PR? Making this PR just adding in remark and the changes it makes.